pom.xml

```
@@ -355,12 +358,12 @@ Export-Package: org.eclipse.emfcloud.jackson.*
 							<dependency>
 								<groupId>org.eclipse.emf</groupId>
 								<artifactId>org.eclipse.emf.ecore.xmi</artifactId>
-								<version>${emf-common-version}</version>
+								<version>2.18.0</version>
```

We should also use a dependency property (add above `<emf-ecore-version>2.18.0</emf-ecore-version>`)
```suggestion
								<version>${emf-ecore-version}</version>
```